cmd/virt-handler/virt-handler.go

```diff
@@ -307,8 +307,9 @@ func (app *virtHandlerApp) Run() {
 	defer close(stop)
 	var capabilities libvirtxml.Caps
 	var hostCpuModel string
+	nodeCapabilitiesVolumePath := "/var/lib/kubevirt-node-capabilities/"
```

Comment:
I personally feel that this PR is already quite big and touches multiple files.
Please consider introducing this change in a follow-up, I don't think its a must in this PR.

Reply:
I don't mind postponing this change to a separate PR.
Any issue with that @iholder101?

Reply:
After deliberation on the matter I've decided to drop the commit for the time being, will postpone it to a separate PR.